examples/scheduler_example/ifplugin/descriptor/intf.go

```diff
@@ -0,0 +1,78 @@
+package descriptor
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ligato/cn-infra/idxmap"
+	. "github.com/ligato/cn-infra/kvscheduler/api"
```

Comment:
I would advice against using dot import, especially here in the implementation. 

You are using 3 dot imports here, with a LOT of brand new code which is not familiar for anyone, but you and using dot imports makes reading this new code really hard since some of the used exported stuff can come from any of those 3 packages.

The generated file should be okay for now, but I would remove the dot imports here for clarity of code.

cc @VladoLavor

Reply:
+1 agree